tsconfig.json

```diff
@@ -30,6 +30,7 @@
     "downlevelIteration": true,
     "outDir": "dist",
     "rootDir": "src",
+    "skipLibCheck": true,
```

Comment:
See https://www.typescriptlang.org/tsconfig#skipLibCheck for details.

>Rather than doing a full check of all d.ts files, TypeScript will type check the code you specifically refer to in your app’s source code.

Otherwise, we get these two errors due to webpack:

```
node_modules/@types/eslint/index.d.ts:451:42 - error TS2724: '"/Users/lukewilson/git/terra/cosmwasm-vm-js/node_modules/@types/estree/index"' has no exported member named 'ChainExpression'. Did you mean 'Expression'?

451         ChainExpression?: ((node: ESTree.ChainExpression & NodeParentExtension) => void) | undefined;
                                             ~~~~~~~~~~~~~~~

node_modules/@types/eslint/index.d.ts:474:43 - error TS2694: Namespace '"/Users/lukewilson/git/terra/cosmwasm-vm-js/node_modules/@types/estree/index"' has no exported member 'ImportExpression'.

474         ImportExpression?: ((node: ESTree.ImportExpression & NodeParentExtension) => void) | undefined;
```